src/betterproto/plugin/models.py

```diff
@@ -360,7 +400,7 @@ def betterproto_field_args(self) -> List[str]:
     def field_wraps(self) -> Optional[str]:
         """Returns betterproto wrapped field type or None."""
         match_wrapper = re.match(
-            r"\.google\.protobuf\.(.+)Value", self.proto_obj.type_name
+            r"\.google\.protobuf\.(.+)Value$", self.proto_obj.type_name
```

Comment:
this was causing chaos by matching `EnumValueDescriptorProto` as if it were `EnumValue`.